pkg/ami/static_resolver.go

```diff
@@ -16,7 +16,11 @@ type StaticDefaultResolver struct {
 func (r *StaticDefaultResolver) Resolve(region, version, instanceType, imageFamily string) (string, error) {
 	logger.Debug("resolving AMI using StaticDefaultResolver for region %s, version %s, instanceType %s and imageFamily %s", region, version, instanceType, imageFamily)
 
-	regionalAMIs := StaticImages[version][imageFamily][ImageClassGeneral]
+	regionalAMIs, ok := StaticImages[version][imageFamily][ImageClassGeneral]
```

Comment:
Do you want make a PR to just add this few lines?

Reply:
I can do it the question is how much it will postpone an already long-running PR